app.yaml

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-application: beacon-go
-version: 1
 runtime: go
 api_version: go1
 
 handlers:
 - url: /.*
   script: _go_app
+
+env_variables:
+#  GOOGLE_CLOUD_PROJECT: change-to-your-project-id
+#  TABLE: genomics-public-data.platinum_genomes.variants
```

Comment:
GOOGLE_BIGQUERY_TABLE ?

Reply:
@anamanolache @kemp-google That's a bit better, thanks.